File: 1. Pattern/Java/11.java
/* package codechef; // don't place package name! */

/*

1 
0 1 
1 0 1 
0 1 0 1 
1 0 1 0 1 

*/
import java.util.*;
import java.lang.*;
import java.io.*;


/* Name of the class has to be "Main" only if the class is public. */
class Codechef
{
    public void nBinaryTriangle(int n) {
        // Write your code here.
        for(int i=0;i<n;i++){
            for(int j=0;j<=i;j++){
                System.out.print((i+j+1)%2+" ");
            }
            System.out.println();
        }
    }
	public static void main (String[] args) throws java.lang.Exception
	{
		// your code goes here
		Codechef obj = new Codechef();
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		obj.nBinaryTriangle(n);
	}
}

File: 1. Pattern/Python/11.py
# 1 
# 0 1 
# 1 0 1 
# 0 1 0 1 
# 1 0 1 0 1 

def nBinaryTriangle(n):
    for i in range(0,n):
        for j in range(0,i+1):
            print((i+j+1)%2,end=" ")
        print("\r")

        
n = int(input())
nBinaryTriangle(n)